ui/pages/Project/constants.js

```diff
@@ -155,8 +154,8 @@ export const FAMILY_FILTER_OPTIONS = [
     value: SHOW_DATA_LOADED,
     category: 'Data Status:',
     name: 'Data Loaded',
-    createFilter: (individualsByGuid, samplesByGuid) => family =>
-      familyVariantSamples(family, individualsByGuid, samplesByGuid).filter(sample => sample.isActive).length > 0,
+    createFilter: (individualsByGuid, user, samplesByFamily) => family =>
+      (samplesByFamily[family.familyGuid] || []).filter(sample => sample.isActive).length > 0,
```

Comment:
the `samplesByFamily` selector will only be run once so these functions can reuse its computation, instead of recomputing the sample list for every single family every time the filter is applied